tutorials/pyroot/pyroot001_arrayInterface.py

```diff
@@ -0,0 +1,46 @@
+## \file
+## \ingroup tutorial_pyroot
+## \notebook -nodraw
+## This tutorial illustrates the conversion of TVec vectors to numpy arrays
```

Comment:
Great Tutorial!!!
Maybe we want to show the other types as well? std::vetor will be most useful!

Reply:
True that it works as well for `std.vector`. I can mention it (or add an example there?).

Reply:
Add it or even remove TVec and mention TVec and the other types. For the moment (and for a while :) ) vector will be more famous than TVec

Reply:
changed the overall scope to `ROOT.std.vector` but mentioned TVec.

Reply:
Changed the overall scope to `ROOT.std.vector` but mentioned TVec